.github/workflows/evm-benchmark.yml

```diff
@@ -0,0 +1,52 @@
+name: EVM benchmark
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - core/vm/**
+  workflow_dispatch:
+    inputs:
+      oldCommit:
+        description: "Old commit hash"
+        required: true
+        types: string
+      newCommit:
+        description: "New commit hash"
+        required: true
+        types: string
+
+concurrency:
+  group: benchmark-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      OLD_COMMIT: ${{ inputs.oldCommit }}
+      NEW_COMMIT: ${{ inputs.newCommit }}
+    steps:
+      - name: 'Checkout Repo'
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c #v3.3.0
+        with:
+          fetch-depth: 0
+
+      - name: 'Setup Go'
+        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 #v3.3.0
+        with:
+          go-version: '1.20.10'
+
+      - name: 'Run EVM benchmark'
+        run: |
+          go install golang.org/x/perf/cmd/benchstat@latest
```

Comment:
we should pin `benchstat` version instead of using latest

Reply:
fixed